vita3k/modules/SceFiber/SceFiber.h

```diff
@@ -19,6 +19,8 @@
 
 #include <module/module.h>
 
+#define SCE_FIBER_CONTEXT_MINIMUM_SIZE 512
```

Comment:
What's this?

Reply:
A constant I used in _sceFiberInitializeImpl and _sceFiberInitializeWithInternalOptionImpl functions to check the size of the context:
```
    if ((sizeContext != 0) && (sizeContext < SCE_FIBER_CONTEXT_MINIMUM_SIZE)) {
        return RET_ERROR(SCE_FIBER_ERROR_RANGE);
    }
```
The size of the context can be equal to 0 or >=SCE_FIBER_CONTEXT_MINIMUM_SIZE